test/CMakeLists.txt

```diff
@@ -178,7 +180,7 @@ set(XSIMD_TESTS
     xsimd_tester.hpp
     xsimd_test_utils.hpp
     xsimd_trigonometric_test.hpp
-    xsimd_trigonometric_test.cpp
+    xsimd_trigonometric_test.cpp]]
 )
 
 add_executable(test_xsimd ${XSIMD_TESTS} ${XSIMD_HEADERS})
```

Comment:
I'm curious: you need to add headers to the executable sources?

Reply:
This allows cmake to detect changes in the header and trigger a new build when you run the make command.

Reply:
Ah, I would have sworn it detected dependencies automatically. My bad.